util/validators/jsonSchema/events.ts

```diff
@@ -0,0 +1,317 @@
+import type {
+  AboutUsUpdate,
+  AboutUsDelete,
+  Donation,
+  Kita,
+  Post,
+  Refugee,
+  Family,
+} from 'util/interfaces/events';
+import { JSONSchemaType } from 'ajv';
+
+export const AboutUsUpdateEvent: JSONSchemaType<AboutUsUpdate> = {
```

Comment:
Um einen Service in der Landingpage anzulegen, braucht man auch zwingend eine URL zu dem Service. Ohne diese wird das Event nicht akzeptiert.